GenderPayGap.Database/Models/Extensions/OrganisationAddress.cs

```diff
@@ -7,75 +7,83 @@
 
 namespace GenderPayGap.Database
 {
-    [Serializable]

```

Comment:
This file's diff looks confusing - because I've moved some code around in the file.

In this file, I've:
* Created a **new** static class `OrganisationAddressExtensions`
* Renamed `GetLines()` to `GetAddressLines()`
* Moved `GetAddressLines()` from `OrganisationAddress` to `OrganisationAddressExtensions` and made it an extension method
  This is to allow us to call `address.GetAddressLines()` where address is null (which happens quite often)